src/android/FileTransferBackground.java

```diff
@@ -91,6 +91,10 @@ public void onServiceConnected(ComponentName componentName, IBinder iBinder) {
             this.managerService = binder.getServiceInstance();
             this.managerService.setMainActivity(cordova.getActivity());
             this.managerService.setConnectedPlugin(this);
+
+            callback(new JSONObject(new HashMap() {{
+                put("state", "INITIALIZED");
```

Comment:
@ombr thoughts ? Ça avait été rajouté pour les tests.

Reply:
For automated tests or for manual testing. If it is for manual testing we should remove it. for automated testing do we need it ?

Reply:
Automated - so we know when the plugin is ready. Otherwise, we have timeouts or we have to wait a bit on each test.

Reply:
Ok let's keep it for now.